### PR TITLE
[DOC] remove asciidoc parts from using guide

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**

Removes asciidoc _parts_ from the Using Guide, as they were not presenting well on the web site.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: documentation/using/master.adoc [hunk 1]
```diff
@@ -25,16 +23,14 @@ include::assemblies/tracing/assembly-distributed-tracing.adoc[leveloffset=+1]
...
 
 [id='api_reference-{context}']
 :parent-context: {context}
 :context: reference
 //schema part
-= Custom resource API reference
+== Custom resource API reference
 
-include::modules/con-common-configuration-properties.adoc[]
-== Schema properties
```
Comment (laidan6000) [inline documentation/using/master.adoc:38]: I did like this heading. Does it need to go?

Comment (PaulRMellor) [reply documentation/using/master.adoc:38]: I didn't think we needed it, but now it's back I think I prefer to have it! One issue, which might not be much of one, is that we now have 4 heading levels when we go down to property descriptions.

image

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

File: documentation/using/master.adoc (hunk 1 above)
Comment (laidan6000) [reply documentation/using/master.adoc:38]: Ah, I didn't consider that. We should avoid any extra nesting of headings. Sorry about that!

Comment (laidan6000) [reply documentation/using/master.adoc:38]: But I don't mind if you want to merge into `master` and see how it looks.

Comment (PaulRMellor) [reply documentation/using/master.adoc:38]: Let's merge it in

**Review by laidan6000 — APPROVED** [review]

**Pull request merged**